spec/custom-gutter-component-spec.coffee

```diff
@@ -0,0 +1,132 @@
+CustomGutterComponent = require '../src/custom-gutter-component'
+Gutter = require '../src/gutter'
+
+describe "CustomGutterComponent", ->
+  [customGutterComponent, gutter] = []
+
+  beforeEach ->
+    mockGutterContainer = {}
+    gutter = new Gutter(mockGutterContainer, {name: 'test-gutter'})
+    customGutterComponent = new CustomGutterComponent({gutter})
+
+  it "creates a gutter DOM node with only an empty 'custom-decorations' child node when it is initialized", ->
+    expect(customGutterComponent.getDomNode().classList.contains('gutter')).toBe true
```

Comment:
On every other component in the text editor, we just have a `domNode` field. Any reason to be different here?